### PR TITLE
New Website: Sofa

### PR DESCRIPTION
**Pull request by TuvalSimha**
- [x] Missing `apiToken ` - @dotansimha

### CODE REVIEW

**Comment by dotansimha** [comment]
regarding tokens: in this repo, only @Urigo can add

**Comment by Urigo** [comment]
Added them all yesterday

**Review by dotansimha — APPROVED** [review]

**Comment by dotansimha** [comment]
@TuvalSimha looks good. feel free to merge and continue with the process.

**Review by ardatan — CHANGES_REQUESTED** [review]
How about the animation?

File: website/public/sitemap.xml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<urlset xmlns="http://www.sitemaps.org/schemas/sitemap/0.9" xmlns:news="http://www.google.com/schemas/sitemap-news/0.9" xmlns:xhtml="http://www.w3.org/1999/xhtml" xmlns:mobile="http://www.google.com/schemas/sitemap-mobile/1.0" xmlns:image="http://www.google.com/schemas/sitemap-image/1.1" xmlns:video="http://www.google.com/schemas/sitemap-video/1.1">
```
Comment (ardatan) [inline website/public/sitemap.xml:2]: Should we git ignore this? @dotansimha

File: website/public/_redirects [hunk 2]
```diff
@@ -0,0 +1 @@
+/docs/introduction /docs 301
```
Comment (ardatan) [inline website/public/_redirects:1]: Also this?
@dotansimha

File: .gitignore [hunk 3]
```diff
@@ -5,3 +5,5 @@ dist/
 npm-debug.log*
 lerna-debug.log
 package-lock.json
+next
```
Comment (B2o5T) [inline .gitignore:8]: what the purpose of this ignore?

File: website/.eslintrc.cjs [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    next: {
+      rootDir: './src',
+    },
+  },
+  rules: {
+    'no-else-return': ['error', { allowElseIf: false }],
+    'react/jsx-curly-brace-presence': ['error', 'never'],
+    'react/jsx-filename-extension': ['error', { extensions: ['.tsx'] }],
+  },
+};
```
Comment (B2o5T) [inline website/.eslintrc.cjs:16]: you don't need eslint config in website

File: website/README.md [hunk 5]
```diff
@@ -0,0 +1,39 @@
...
+
+This command generates static content into the `build` directory and can be served using any static contents hosting service.
+
+### Deployment
+
+```
+GIT_USER=<YOUR_GITHUB_USERNAME> USE_SSH=true yarn deploy
+```
+
+If you are using GitHub pages for hosting, this command is a convenient way to build the website and push to the `gh-pages` branch.
```
Comment (B2o5T) [inline website/README.md:39]: from where you copied this readme? just remove it

File: website/next.config.mjs [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+    ignoreDuringBuilds: true,
+  },
+  webpack(config, meta) {
+    applyUnderscoreRedirects(config, meta);
+
+    return config;
+  },
+  redirects: () =>
+    Object.entries({
+      '/docs/introduction': '/docs',
```
Comment (B2o5T) [inline website/next.config.mjs:24]: this redirect is from graphql-config

File: website/package.json [hunk 7]
```diff
@@ -1,16 +1,36 @@
...
-    "build": "docusaurus-build",
-    "version": "docusaurus-version",
-    "rename-version": "docusaurus-rename-version",
-    "build:animation": "rm -rf static/animation && cd animation && yarn && npm run build && cp -r ./build ../static/animation",
-    "predeploy": "yarn build:animation",
-    "deploy": "docusaurus-publish"
+    "algolia-sync": "node scripts/algolia-ci.mjs",
+    "build:animation": "rm -rf public/animation && cd animation && yarn && npm run build && cp -r ./build ../public/animation",
+    "prebuild": "npm run build:animation",
+    "build": "next build && next-sitemap --config next-sitemap.config.cjs ",
```
Comment (B2o5T) [inline website/package.json:10]:
```suggestion
    "build": "next build && next-sitemap --config next-sitemap.config.cjs",
```

File: website/package.json [hunk 8]
```diff
@@ -1,16 +1,36 @@
...
+    "predev": "npm run build:animation",
+    "dev": "next dev",
+    "next": "next",
```
Comment (B2o5T) [inline website/package.json:13]:
```suggestion
    "dev": "next",
```

File: website/scripts/algolia-ci.mjs [hunk 9]
```diff
@@ -0,0 +1,15 @@
+import {resolve, dirname} from 'node:path';
+import {fileURLToPath} from 'node:url';
+import {indexToAlgolia} from '@guild-docs/algolia';
```
Comment (B2o5T) [inline website/scripts/algolia-ci.mjs:3]:
```suggestion
import {indexToAlgolia} from '@theguild/algolia';
```

Comment (B2o5T) [reply website/scripts/algolia-ci.mjs:3]: and remove old package and instal new one

File: website/src/pages/meta.json [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+    "theme": {
+      "layout": "raw"
+    }
+  },
+  "docs": {
+    "title": "Docs",
+    "type": "page",
+    "theme": {
+      "toc": true
+    }
```
Comment (B2o5T) [inline website/src/pages/meta.json:14]:
```suggestion
    "type": "page"
```

File: website/public/sitemap.xml (hunk 1 above)
Comment (TuvalSimha) [reply website/public/sitemap.xml:2]: Oh yeah, i will delete it

File: website/public/_redirects (hunk 2 above)
Comment (TuvalSimha) [reply website/public/_redirects:1]: The same

File: .gitignore (hunk 3 above)
Comment (TuvalSimha) [reply .gitignore:8]: Deleted :)

File: website/.eslintrc.cjs (hunk 4 above)
Comment (TuvalSimha) [reply website/.eslintrc.cjs:16]: Deleted :)

File: website/theme.config.tsx [hunk 11]
```diff
@@ -1,41 +1,77 @@
...
+  ModulesLogo,
+  FooterExtended,
+  mdxComponents,
+  Header,
+  Navbar,
+} from '@theguild/components';
+import { DocsThemeConfig } from 'nextra-theme-docs';
 
-const SITE_NAME = 'Sofa';
+const SITE_NAME = 'GraphQL Modules';
```
Comment (B2o5T) [inline website/theme.config.tsx]: sofa

**Pull request merged**